submit.py

```diff
@@ -152,7 +158,7 @@ def get_url(cfg, option, default):
 def get_config():
     """Returns a ConfigParser object for the .kattisrc file(s)
     """
-    cfg = configparser.ConfigParser()
+    cfg = ConfigParser()
```

Comment:
I think the other version, with `import ConfigParser as configparser`, is nicer.